### PR TITLE
Add TLS configuration instructions

### PR DESCRIPTION
**Pull request by himanchali**
Doc to add TLS parameters in the Aerospike JDBC URL with some tested examples

### CODE REVIEW

File: docs/tls.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Aerospike JDBC TLS Configuration Parameters
+
+The Aerospike JDBC driver can be configured to use SSL through the JDBC connection URL. 
+
+Below are the tls-related configs for aerospike-jdbc (example with value):
+```tlsEnabled true
```
Comment (reugn) [inline docs/tls.md:6]: New line after ```

File: docs/tls.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+tlsName TLS_NAME
+tlsStoreType (like jks)
+tlsTruststorePassword.   ******* (if required)
+tlsTruststorePath.  ./../XXX.jks (the full path)
+```
+The full list of the valid values can be found at [AerospikeTLSPolicyConfig](https://github.com/aerospike/aerospike-jdbc/blob/main/src/main/java/com/aerospike/jdbc/tls/AerospikeTLSPolicyConfig.java).
+
+The configuration expects a standard Java truststore. The example to add CA certificate to Java Truststore can be found at [Add CA certificate to Java TrustStore on client nodes](https://docs.aerospike.com/server/operations/configure/network/tls/mtls_java#add-ca-certificate-to-java-truststore-on-client-nodes).
+
+```keytool -import -alias tls1 -file /etc/aerospike/ssl/tls1/cert.pem -keystore //usr/lib/jvm/java-8-openjdk-amd64/jre/lib/security/cacerts -storePass changeit
```
Comment (reugn) [inline docs/tls.md:16]: New line after ```

File: docs/tls.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+```
+Here are some working URL example using the above configs:
+
+```//aerospike server detail
```
Comment (reugn) [inline docs/tls.md:20]: New line after ```

File: docs/tls.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Aerospike JDBC TLS Configuration Parameters
+
+The Aerospike JDBC driver can be configured to use SSL through the JDBC connection URL. 
+
+Below are the tls-related configs for aerospike-jdbc (example with value):
+```tlsEnabled true
+tlsName TLS_NAME
+tlsStoreType (like jks)
+tlsTruststorePassword.   ******* (if required)
+tlsTruststorePath.  ./../XXX.jks (the full path)
```
Comment (reugn) [inline docs/tls.md:10]:
```suggestion
tlsTruststorePassword   ******* (if required)
tlsTruststorePath  ./../XXX.jks (the full path)
```

File: docs/tls.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+```
+The full list of the valid values can be found at [AerospikeTLSPolicyConfig](https://github.com/aerospike/aerospike-jdbc/blob/main/src/main/java/com/aerospike/jdbc/tls/AerospikeTLSPolicyConfig.java).
+
+The configuration expects a standard Java truststore. The example to add CA certificate to Java Truststore can be found at [Add CA certificate to Java TrustStore on client nodes](https://docs.aerospike.com/server/operations/configure/network/tls/mtls_java#add-ca-certificate-to-java-truststore-on-client-nodes).
```
Comment (reugn) [inline docs/tls.md:14]:
```suggestion
The configuration expects a standard Java truststore. An example of adding a CA certificate to the Java Truststore can be found at [Add CA certificate to Java TrustStore on client nodes](https://docs.aerospike.com/server/operations/configure/network/tls/mtls_java#add-ca-certificate-to-java-truststore-on-client-nodes).
```

File: docs/tls.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+```keytool -import -alias tls1 -file /etc/aerospike/ssl/tls1/cert.pem -keystore //usr/lib/jvm/java-8-openjdk-amd64/jre/lib/security/cacerts -storePass changeit
+```
+Here are some working URL example using the above configs:
```
Comment (reugn) [inline docs/tls.md:18]:
```suggestion
Here are some working URL examples using the above configuration properties:
```

File: docs/tls.md (hunk 1 above)
Comment (himanchali) [reply docs/tls.md:7]: have added newline before and after the code block

File: docs/tls.md (hunk 2 above)
Comment (himanchali) [reply docs/tls.md]: have added new line before and after the code block

File: docs/tls.md (hunk 3 above)
Comment (himanchali) [reply docs/tls.md:21]: have added new line before and after the code block

**Review by reugn — APPROVED** [review]

**Pull request merged**